crates/samples/windows/enum_windows/src/main.rs

```diff
@@ -13,8 +15,14 @@ extern "system" fn enum_window(window: HWND, _: LPARAM) -> BOOL {
         let len = GetWindowTextW(window, &mut text);
         let text = String::from_utf16_lossy(&text[..len as usize]);
 
-        if !text.is_empty() {
-            println!("{text}");
+        let mut info = WINDOWINFO {
+            cbSize: core::mem::size_of::<WINDOWINFO>() as u32,
+            ..Default::default()
+        };
+        GetWindowInfo(window, &mut info).unwrap();
+
+        if !text.is_empty() && info.dwStyle & WS_VISIBLE.0 != 0 {
```

Comment:
Looks like `WINDOWINFO`'s `dwStyle` and `dwExStyle` fields need to be adjusted to use the `WINDOW_STYLE` and `WINDOW_EX_STYLE` types. Could you open an issue for that? https://github.com/microsoft/win32metadata

Reply:
No problem. https://github.com/microsoft/win32metadata/issues/1430